raspberry/setup.py

```diff
@@ -36,10 +36,7 @@
           exclude=["tests/"]),
       include_package_data=True,
       install_requires=[
-          'smbus', 'pigpio', 'bitarray', 'pyserial', 'VL53L0X', 'pandas', 'numpy', 'matplotlib', 'wheel', 'control'
-      ],
-      dependency_links=[
-          "git+https://github.com/pimoroni/VL53L0X-python.git",
+          'smbus', 'bitarray', 'pyserial', 'VL53L0X @ git+https://github.com/pimoroni/VL53L0X-python.git', 'pandas', 'numpy', 'matplotlib', 'wheel', 'control'
```

Comment:
Do we really need wheel and matplotlib and control? Note that both of these are quite heavy packages and we should not be plotting anything on the quadcopter.

Reply:
think we should sort this in PR #146 first